### PR TITLE
Connect to eth

### LINKED ISSUE #27: Setup connection between Ethereum and the FE

### LINKED ISSUE #28: Remove `compile.js` and `deploy.js`
Once Truffle is fully implemented, we will no longer need these 2 files.

### LINKED ISSUE #30: Create documentation for truffle env setup

### PR DESCRIPTION
**Pull request by jpc20**
# Pull request checklist

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [x] Other (please describe): Established a connection between our Vue app and our contracts on the Rinkeby test network.

## Current behavior

- We had no connection between the blockchain and our Vue app up until this point.

## New behavior

- Rewrote the `factory.js` and `campaign.js` files and tested the connections in a console.log from the home page.

## Other information

- Added documentation for setting up Truffle and Ganache.

closes #27  
closes #28 
closes #30

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -7,6 +7,15 @@
...
 import CampaignContainer from '@/components/Container.vue';
+import factory from '../../ethereum/factory'
+import campaign from '../../ethereum/campaign'
+
+factory.at('0x7c70286f6991c660a0cC6d52A74aEBbDE45Da380')
+  .then((instance) => instance.getDeployedCampaigns())
+  .then((campaignAddresses) => console.log(campaignAddresses))
+
+campaign.at("0xe8a0980C2B37C2C4FCE45e579301B00CC824bCFc")
+  .then((instance) => console.log(instance))
```
Comment (jpc20) [inline src/views/Home.vue:18]: We will move these when it is time to set up the real interactions, but I just wanted to give an idea of how they will look.

File: yarn.lock [hunk 2]
```diff
@@ -2484,11 +2484,6 @@ anymatch@~3.1.1:
     normalize-path "^3.0.0"
     picomatch "^2.0.4"
 
-app-module-path@^2.2.0:
```
Comment (jpc20) [inline yarn.lock:2487]: Not sure why all of this got deleted... still getting used to yarn.

Comment (edmdc) [reply yarn.lock:2487]: Opps, probably happened when we last merged main. Once the project matures, the dependencies will stabilize.

**Review by edmdc — APPROVED** [review]
Good to see the connection to the blockchain. Let's take a moment during the following days to standardize a schema of blockchain interactions, so each team is clear on how exactly they are to interact with it.

**Review by attom2 — APPROVED** [review]
Looks good, I'm interested to see how we handle new requests

**Pull request merged**